rules/manage-urges/rule.md

```diff
@@ -0,0 +1,68 @@
+---
+seoDescription: Emotional reactions are natural, but managing them is key. Learn the 'Pause and Plan' technique to handle your emotions effectively.
+type: rule
+archivedreason: null
+title: Do you know how to manage urges?
+guid: b3728da1-c02a-4f3b-9346-08c6f720e173
+uri: manage-urges
+created: 2024-08-08T18:48:19.0000000Z
+authors:
+  - title: Adam Cogan
+    url: https://ssw.com.au/people/adam-cogan
+  - title: Brady Stroud
+    url: https://ssw.com.au/people/brady-stroud
+related: 
+  - accepting-unsolicited-feedback
+  - how-to-respond-to-criticism
+redirects: []
+---
+
+Emails or conversations can sometimes trigger a poor reaction, leading to regrets. Managing your urges helps you overcome bad habits, avoid emotional outbursts, and make better decisions.
```

Comment:
Please change from "a poor reaction" to "negative emotional reactions".

I feel that "poor reaction" is a bit vague